index.html

```diff
@@ -8,21 +8,39 @@
     <title>Portfolio - A description of O-va's profile</title>
 </head>
 <body>
+    
     <div class="body-wrapper">
-        <section class="header">
-            <ul class="nav" id="nav">
-                <li><h5 id="logo">O-va</h5></li>
-                <li class="portfolio"><a href="portfolio.html">Portfolio</a></li>
-                <li class="about"><a href="about.html">About</a></li>
-                <li class="contact"><a href="contact.html">Contact</a></li>
-                <li class="email"><a href="contact.html"><img src="./assets/email.png" alt="email"></a></li>
-                <li>
-                    <span>
-                        <img id="union" src="./assets/Union.png" alt='union-sign'/>
-                    </span>
-                </li>
-            </ul>
-        </section>
+
+
+                    
+            <nav class="nav" id="nav">
+                <div class="logo_header">  
+                  <a href="#" class="logo">Ova</a>
+                  <div class="navbar_toggle">
+                    <div class="bar1"></div>
+                    <div class="bar2"></div>
+                    <div class="bar3"></div>
+                  </div>
+                </div>
+                
+                <ul class="main_nav">
+                  <li>
+                    <a href="#work-flex" class="nav_link1">Portfolio</a>
+                  </li>
+                  <li>
+                    <a href="#hero" class="nav_link2">About</a>
+                  </li>
+                  <li>
+                    <a href="#form" class="nav_link3">Contact</a>
+                  </li>
+                  <li>
```

Comment:
- [ ] kindly align your links so they follow the figma design. your close icon should be well positioned also. please make fix that.
your version
![Screenshot (271)_LI](https://user-images.githubusercontent.com/65298048/181122280-222beda1-ab5a-4897-b90e-fb66e32919f6.jpg)
figma version
![Screenshot (270)](https://user-images.githubusercontent.com/65298048/181122307-6fbd7421-c008-43fa-b97e-f91c8c2f102d.png)

Reply:
@tufoinnkuo10 I have aligned the elements according to the figma template.